### PR TITLE
Add basic CI/CD

### PR DESCRIPTION
**Pull request by snovikov**

### CODE REVIEW

**Review by titanlien — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+.PHONY: help gen lint test _gen-main _gen-examples _gen-modules _lint-files _lint-fmt _lint-json _pull-tf _pull-tfdocs _pull-fl _pull-jl
+
+CURRENT_DIR = $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+TF_EXAMPLES = $(sort $(dir $(wildcard $(CURRENT_DIR)examples/*/)))
+TF_MODULES  = $(sort $(dir $(wildcard $(CURRENT_DIR)modules/*/)))
+
+# -------------------------------------------------------------------------------------------------
+# Container versions
+# -------------------------------------------------------------------------------------------------
+TF_VERSION      = 0.13.7
```
Comment (danvaida) [inline Makefile:14]: This looks a little bit too old, no?

Comment (snovikov) [reply Makefile:14]: For back compatability.

Comment (snovikov) [reply Makefile:14]: If we are going to make it open source - I think it should be OK

**Pull request merged**